ranger-plugin/scripts/enable-ranger-plugin.sh

```diff
@@ -237,9 +305,13 @@ main() {
   if [[ $# -lt 1 ]]; then
     usage
   fi
+  if [[ -z "$GPHOME" ]]; then
+    GPHOME="/usr/local/hawq"
+  fi
   SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd -P)"
   parse_params "$@"
   validate_params
+  sync_rps_configuration
```

Comment:
if sync fails, Ranger definitions are not created, which is not good. Service can work without standby and without sync, so either not fail (just warn) or move it to the very last step. Better yet, not to sync at all here, as I commented above.